csharp/ql/src/semmle/code/csharp/controlflow/internal/pressa/SsaImplCommon.qll

```diff
@@ -351,8 +360,7 @@ private module SsaDefReaches {
    */
   predicate defAdjacentRead(Definition def, BasicBlock bb1, BasicBlock bb2, int i2) {
     varBlockReaches(def, bb1, bb2) and
-    ssaRefRank(bb2, i2, def.getSourceVariable(), SsaRead()) = 1 and
-    variableRead(bb2, i2, _, _)
```

Comment:
Why was `variableRead(bb2, i2, _, _)` removed?

Reply:
Because it is implied by `ssaRefRank(bb2, i2, def.getSourceVariable(), SsaRead())`, so it was simply redundant.